### DISCUSSION TITLE
Cannot destructure property `isOnChange` error when wrapping Controller in render() @testing-library/react and jest

### DISCUSSION THREAD

**Discussion by lsbyerley** [react-hook-form/react-hook-form#2297] [Q&A] [answered]
Hello everyone, as anyone ever ran into this error when attempting to write tests for a component that uses the `Controller` component? I am using the `render()` function from the react testing library and when running the test with jest I get the below error message.

`TypeError: Cannot destructure property 'isOnChange' of 'undefined' or 'null'.` 

I realize a codesandbox is needed for this, but I wanted to get this out there and I will post an update to the codesandbox link. Below is an image of where the error is pointing too and a below the image is how i'm using the `Controller` component.

<img width="794" alt="Screen Shot 2020-07-20 at 4 27 23 PM" src="https://user-images.githubusercontent.com/3066258/87983213-02e27980-caa6-11ea-99bc-6f7be6611a65.png">

    <Controller
      name="body.markup"
      control={control}
      defaultValue={defaultValue}
      render={({ onChange, value, label }) => <Froala onChange={onChange} value={value} brandPlugins={brandPlugins} />}
    />

**Comment by bluebill1049** [comment]
are you testing the component with `useForm`'s `control`?

**Comment by lsbyerley** [comment, marked as answer]
Turns out I needed to wrap my component in react-hook-forms Provider component in the test. Closing this issue
